.secrets-example.json

```diff
@@ -0,0 +1,4 @@
+{
+    "PANDAOPS_CLIENT_ID":"xyz",
+    "PANDAOPS_CLIENT_SECRET":"abc"
+}
```

Comment:
I don't think we need this file, it might just be confusing. The json example you gave in the `README` should be more than enough to figure out what's going on

Reply:
I have seen projects which do it this way. I like it because then I can just do ``cp .secrets-example.json .secrets.json``, but sure I can remove it

Reply:
Gotcha, that makes sense. Ehh, I don't think it's a significant decision either way, so I'm just going to go ahead with the merge